aten/src/ATen/native/nested/cuda/NestedTensorTransformerFunctions.cpp

```diff
@@ -202,5 +205,36 @@ Tensor NestedTensor_to_padded_tensor_cuda(
   }
   return NestedTensor_to_padded_tensor_generic(t, padding, output_size);
 }
+
+Tensor flash_scaled_dot_product_self_attention(
+    const Tensor& qkv,
+    const Tensor& cumulative_sequence_length,
+    const int64_t max_seqlen_batch,
+    double dropout_p,
+    bool causal) {
+#if defined(CUDA_VERSION) && CUDA_VERSION >= 11060 && !defined(_MSC_VER) && \
+    !(__APPLE__ || __MACH__)
+  auto q = qkv.index({at::indexing::Slice(), 0});
```

Comment:
Does this allow passing in Q with different seqlen to K/V? Specific to incremental decoding for decoder models.